### PR TITLE
Create hosts and evidence

### PR DESCRIPTION
**Pull request by rachkor**
This PR adds an Evidence template to the Plugin Manager so the Request/Response data can be added into instances of Evidence rather than creating duplicate Issues.

### CODE REVIEW

File: lib/dradis/plugins/netsparker/importer.rb [hunk 1]
```diff
@@ -19,20 +19,35 @@ def import(params={})
       end
 
       @doc.xpath('/netsparker/vulnerability').each do |xml_vuln|
-        process_vulnerability(xml_vuln)
```
Comment (etdsoft) [inline lib/dradis/plugins/netsparker/importer.rb]: I liked the idea of moving the responsibility of understanding how to parse the vuln data into a separate method.

File: lib/dradis/plugins/netsparker/importer.rb [hunk 2]
```diff
@@ -19,20 +19,35 @@ def import(params={})
...
+        # Create Nodes from the <url> tags
```
Comment (etdsoft) [inline lib/dradis/plugins/netsparker/importer.rb]: If in the future we're going to want to have all plugins create nodes by IP address, maybe we should start doing this for new plugins like this one?

I'm OK with what we have, but it seems that in the future we'd like the Node's label to be the IP address, and a node "hostname" property to be the value of the host from the `./url` element.

Comment (rachkor) [inline lib/dradis/plugins/netsparker/importer.rb]: @etdsoft, unfortunately, it doesn't look like Netsparker includes the IP address in the XML output. I checked all the sample files I have and none of them include an IP ☹️

File: lib/dradis/plugins/netsparker/importer.rb (hunk 1 above)
Comment (rachkor) [inline lib/dradis/plugins/netsparker/importer.rb]: @etdsoft 👍 pushed!

**Pull request merged**